games-simulation/simutrans_pak128/simutrans_pak128-2.6.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Pakset for Simutrans"
+DESCRIPTION="A 'pakset' is the data set that Simutrans needs to run: \
+it contains all the data for the graphics of individual items (vehicles, \
+roads, railways, buildings, etc.) and all of the balancing information \
+(such as prices, speeds, capacities and so forth) necessary to make \
+the game complete.
+When Simutrans could only support 64px size graphics, pak128 already \
+started. First pak to feature a complex economy and have a very wide \
+variety of objects. It contains roughly 7 time more graphics data than \
+pak64 and thus requires by far the largest amount of RAM and \
+processing power of all Simutrans sets."
+HOMEPAGE="http://www.simutrans.com"
+SOURCE_URI="https://downloads.sourceforge.net/project/simutrans/pak128/pak128%20for%20ST%20120%20(2.6%2C%20completed%20elevated%20tracks)/pak128-2.6--ST120.zip"
+CHECKSUM_SHA256="ac0835ff667f80a2990ae226e8a58a9c44bbd1c2bf4a9f0f3cd87614ef9da94f"
+SOURCE_FILENAME="simupak_128-$portVersion.zip"
+SOURCE_DIR=""
+COPYRIGHT="2004-2007 Tomas Kubes
+	2008-2016 The pak128-Team"
+LICENSE="Artistic"
+REVISION="1"
+ARCHITECTURES="any"
+
+PROVIDES="
+	simutrans_pak128 = $portVersion
+	simutrans_pakset = 120.2 compat >= 120.2
+	"
+
+GLOBAL_WRITABLE_FILES="
+	non-packaged/$relativeAppsDir/simutrans/pak128/config directory keep-old
```

Comment:
@pulkomandy 
I've updated the pull request to remove the fixPkgconfig entry.

However, it seems not trivial to go from `/non-packaged/$relativeAppsDir` to `$relativeSettingsDir`. While this still works for packages installed in the system hierarchy (config directory is copied to `/system/settings/simutrans/config` and `/apps/simutrans/config` links to it), I can't get it to work in the /home/config hierarchy because `/home/config/settings/simutrans/config` is not created at all, so the link from `/home/config/apps/simutrans/config` is broken. Using non-packaged for the config directory on the other hand works for both installation in system and in home.